src/views/tutorial-view/utils/get-collection-context.ts

```diff
@@ -25,6 +26,13 @@ interface CurrentCollectionTutorial {
   }
 }
 
+const moizeOpts: Options = {
+  isPromise: true,
+  maxSize: Infinity,
```

Comment:
Any particular reason to not use `isDeepEqual: true,` here, but use it in the other 2 spots?

Reply:
In this case we only pass a string as an arg to this call, as opposed to an object with nested keys (with `getAllCollections`) where we need the deep equality check.